readme.txt

```diff
@@ -3,7 +3,7 @@ Contributors: nathanrice, studiopress, wpmuguru
 Donate link: https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=5553118
 Tags: tabs, ui-tabs, genesis, genesiswp, studiopress
 Requires at least: 3.7
-Tested up to: 4.8.3
+Tested up to: 5.2.2
 Stable tag: 1.0.0
```

Comment:
Hey, @ethan-schaffer we'd need to update the version here as well, so it does reflect that the stable version of this plugin is the current one.

```suggestion
Stable tag: 1.0.1
```

The same would need to be done [here](https://github.com/studiopress/genesis-tabs/blob/release/1.0.1/plugin.php#L10) and [here](https://github.com/studiopress/genesis-tabs/blob/release/1.0.1/plugin.php#L44)

Once these lines are updated, we're good to merge :)